%YAML:1.0
---
world_rvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ -1.8218753784627399e+00, -6.3578182026423347e-01,
       -1.5330925233901831e+00 ]
world_tvec: !!opencv-matrix
   rows: 3
   cols: 1
   dt: d
   data: [ 1.4687971368414168e+00, -4.7638966983549560e+00,
       -7.8054160830375370e+00 ]
